packages/cli/src/lib/cmds/build.ts

```diff
@@ -55,6 +57,20 @@ class Build {
       private prompt: Prompt = new InquirerPrompt()) {
   }
 
+  /**
+   * Checks if the twa-manifest.json file has been changed since the last time the project was generated.
+   */
+  async checkManifest(manifestFile: string): Promise<boolean> {
+    const manifestContents = fs.readFileSync(manifestFile);
+    const checksumFile = path.join(process.cwd(), 'manifest-checksum.txt');
+    const prevChecksum = (await fs.promises.readFile(checksumFile)).toString();
+    const currChecksum = crypto.createHash('sha1').update(manifestContents).digest('hex');
+    if (currChecksum != prevChecksum) {
+      return true;
+    }
+    return false;
```

Comment:
These lines can just be `return currChecksum != prevChecksum`.

Reply:
done.